types/prosemirror-model/index.d.ts

```diff
@@ -877,6 +877,11 @@ export class ResolvedPos<S extends Schema = any> {
    * before the position is returned.
    */
   nodeBefore?: ProsemirrorNode<S> | null;
+  /**
+   * Get the position at the given index in the parent node at the
+   * given depth (which defaults to this.depth).
+   */
+  posAtIndex(index: number, depth?: number): number;
```

Comment:
It looks like this was added later than the listed version at the top of the file. This was published in prosemirror 1.11, but the comment on the top of the file reads `// Type definitions for prosemirror-model 1.7`. That should be updated to 1.11 before this can be merged.

Reply:
done